tests/journald_tests.cpp

```diff
@@ -270,15 +290,30 @@ TEST_F(JournaldLoggerTest, ROOT_LogToJournald)
       statusRunning->executor_id().value(),
       "runs",
       "latest");
-
   ASSERT_TRUE(os::exists(sandboxDirectory));
 
-  string stdoutPath = path::join(sandboxDirectory, "stdout");
-  ASSERT_TRUE(os::exists(stdoutPath));
+  std::string stdoutPath = path::join(sandboxDirectory, "stdout");
+
+  if (GetParam() == "journald") {
+    // Check that the sandbox was *not* written to.
+    // TODO(josephw): This file exists as other parts of the agent will
+    // create it.  We should invert this assertion when we fix this in Mesos.
+    ASSERT_TRUE(os::exists(stdoutPath));
 
-  Result<string> stdout = os::read(stdoutPath);
-  ASSERT_SOME(stdout);
-  EXPECT_TRUE(strings::contains(stdout.get(), specialString));
+    Result<std::string> stdout = os::read(stdoutPath);
+    ASSERT_SOME(stdout);
+    EXPECT_FALSE(strings::contains(stdout.get(), specialString));
```

Comment:
It should be empty, but in case we have extremely verbose logging enabled (in the command to run the tests), the file may be non-empty, even with nothing to fetch.